lib/git/status.go

```diff
@@ -4,9 +4,16 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/Originate/git-town/lib/config"
 	"github.com/Originate/git-town/lib/util"
 )
 
+func EnsureDoesNotHaveUncommitedChanges(message string) {
+	if HasOpenChanges() {
```

Comment:
Let's standardize on either "uncommitted changes" or "open changes". The two methods on this line and the one above use different terminology for the same thing.